src/material-experimental/mdc-slider/_slider-theme.scss

```diff
@@ -110,4 +110,10 @@
       base: mdc-theme-prop-value($color)
     ),
   ));
+  .mdc-slider-hover-ripple {
```

Comment:
Should we document where we get these opacity values? Is it possible to grab em from MDC?

Reply:
As far as I'm aware, no. They don't seem to have a specific opacity for their ripples. I was just visually following the [MDC Slider Spec](https://material.io/components/sliders)

I did see that mdc does define [a ripple opacity var for the mdc-radio](https://github.com/material-components/material-components-web/blob/master/packages/mdc-radio/_radio-theme.scss#L36), which we use [in our implementation](https://github.com/angular/components/blob/master/src/material-experimental/mdc-radio/radio.scss#L20). Should I create an issue for them to do this

Reply:
Yeah, if they could expose the opacity then that would be helpful for us and one less thing we need to maintain

Reply:
I created [Issue #6876](https://github.com/material-components/material-components-web/issues/6876) requesting they fix their current ripple implementation (they are missing the "pressed" state) and then expose the 'hover', 'focused', and 'pressed' ripple opacities